alacritty_terminal/src/term/mod.rs

```diff
@@ -815,14 +815,17 @@ pub struct Term<T> {
     input_needs_wrap: bool,
 
     /// Alternate grid.
-    alt_grid: Grid<Cell>,
+    inactive_grid: Grid<Cell>,
 
     /// Alt is active.
     alt: bool,
```

Comment:
Is this related to the the `alt_grid` field? If so, should this also undergo a rename?

Reply:
This is actually the crux of the confusion. The idea is that `alt` field indicates if we're in alt mode, and should probably be removed in favor of checking that directly. See https://github.com/alacritty/alacritty/pull/3510#issuecomment-605546448 and https://github.com/alacritty/alacritty/pull/3510#discussion_r397043039 for more about this.

Reply:
I see, I missed those. Thanks for clarifying!